backend/app/controllers/spree/admin/refunds_controller.rb

```diff
@@ -25,6 +40,10 @@ def refund_reasons
         @refund_reasons ||= Spree::RefundReason.active.all
       end
 
+      def refund_params
+        params.require(:refund).permit!
```

Comment:
Is there a reason we are not explicitly whitelisting parameters here?

Reply:
I don't think we are whitelisting params in the admin for any controller. It's how we do in [ResourceController](https://github.com/solidusio/solidus/blob/ac325ef25194a61bf97a7ad975cd50853ac921ce/backend/app/controllers/spree/admin/resource_controller.rb#L281), which is inherited by all the other controllers.